### ISSUE TITLE
Address the Dependabot alerts for Che-Theia repository

### ISSUE THREAD

**Issue by azatsarynnyy** [eclipse/che#21034]
### Is your task related to a problem? Please describe

There're 82 alerts reported by Dependabot for the Che-Theia repository.
https://github.com/eclipse-che/che-theia/security/dependabot?page=1&q=is%3Aopen

### Describe the solution you'd like

Need to look at how we can fix the security issues considering that we're pinned to the specific commit SHA from upstream Theia.

**Comment by RomanNikitenko** [comment]
I'm investigating the alerts reported by Dependabot.
38 alerts were fixed by PRs which you can see above.

I'm going to post bellow info about some dependencies which we can not fix/upgrade right now.

`simple-get`
@theia/core  => keytar  => prebuild-install  => simple-get
So, `keytar`dependency should be updated in upstream to upgrade `simple-get` version.

**Comment by RomanNikitenko** [comment]
The main steps which could help to reduce `Dependabot` alerts are:
- upgrade `che-theia` to the latest `theia` version
- upgrade `lerna` version (at the moment using `lerna` version >= 3 causes problems related building `che-theia` assembly)
- upgrade some versions in `theia/plugin-packager` (please see the corresponding info in the comment above) to upgrade transitive dependencies to non-vulnerable versions.
- find alternative to `vuepress` as the `vuepress` brings some vulnerable versions of transitive dependencies

**Comment by RomanNikitenko** [comment]
Within current issue I reduced number of the `Dependabot` alerts from 82 to 12.
In the comment above I described steps which can help to improve the current state, so additional efforts are required.

My propose is - close the current issue and come back to the `Dependabot` alerts when we:
- provide alignment to the latest upstream version
- resolve some problems related to building `che-theia` assembly with `lerna` >=3

@azatsarynnyy 
wdyt?
does it look reasonable from your point of view?
